doc/getting-started/getting-started.rst

```diff
@@ -506,7 +506,7 @@ Build ACRN
    .. code-block:: bash
 
       cd ~/acrn-work/acrn-hypervisor
-      make -j $(nproc) BOARD=~/acrn-work/my_board.xml SCENARIO=~/acrn-work/industry.xml
+      make -j $(nproc) BOARD=~/acrn-work/my_board.xml SCENARIO=~/acrn-work/shared.xml
       make targz-pkg
 
```

Comment:
Some GSG images of the configurator show the URL at the top of the screen. The word "industry" appears in the URL. Recommend cropping out the entire URL area, as it doesn't add much value. Affected images: 
- gsg_config_launch_generate.png 
- gsg_config_launch_load.png
- gsg_config_scenario_load.png 
- gsg_config_scenario_save.png

Reply:
Doh. Will do. Thanks

Reply:
I trimmed the URL/tool bar off all the ``gsg_config*.png`` images to keep them consistent.